apps/docs/src/app/platform/component-docs/platform-search-field/platform-search-field.module.ts

```diff
@@ -30,7 +33,9 @@ const routes: Routes = [
         PlatformSearchFieldHeaderComponent,
         PlatformSearchFieldBasicExampleComponent,
         PlatformSearchFieldCategoriesExampleComponent,
-        PlatformSearchFieldDataSourceExampleComponent
-    ]
+        PlatformSearchFieldDataSourceExampleComponent,
+        PlatformSearchFieldMobileExampleComponent
+    ],
+    providers: [{ provide: MOBILE_MODE_CONFIG, useValue: SEARCH_FIELD_MOBILE_CONFIG, multi: true }],
```

Comment:
We're not providing config in the components' modules. It should be provided in Docs modules.